std/machines/binary.asm

```diff
@@ -8,9 +8,6 @@ machine Binary with
     // Allow this machine to be connected via a permutation
     call_selectors: sel,
 {
-
-    // lower bound degree is 262144
```

Comment:
I'm counting 196608 - am I doing something wrong?

Reply:
This one I really thought was 2**18, but yea also don't see why. Maybe next power of 2